stlab/expected.hpp

```diff
@@ -1,9 +1,11 @@
 /*
-Copyright 2016 Adobe
```

Comment:
Unfortunately the corpyright must remain Adobe's or I can't accept the pull request (Adobe has been burned several times by other parties changing the licensing requirements, leaving a technology orphaned).